.pre-commit-config.yaml

```diff
@@ -4,30 +4,30 @@
 exclude: (profilers|sandbox|\.chatgpt)
```

Comment:
```suggestion
```

While we're in here lets remove these artifacts.

Reply:
Done: 59839f9